src/components/smart_objects/src/smart_object.cc

```diff
@@ -298,6 +290,16 @@ bool SmartObject::operator==(const int64_t Value) const {
 }
 
 // =============================================================
+// uint64_t TYPE SUPPORT
```

Comment:
@malirod These comments are in the whole file, should I remove only this or other too?